src/modules/link.cpp

```diff
@@ -1,22 +1,119 @@
 //
 // Link Module
 //
-
+#include <Arduino.h>
```

Comment:
For using data types like uint32_t instead of unsigned long to make the structure easier to review byte mapping this include was required with Arduino.

Reply:
Shouldn't include Arduino.h -- use stdint.h for `uint32_t`/similar data types